pkg/cosign/tlog.go

```diff
@@ -41,6 +44,19 @@ import (
 // rekor.pub should be updated whenever the Rekor public key is rotated & the bundle annotation should be up-versioned
 //go:embed rekor.pub
 var rekorPub string
+var rekorTargetStr = `rekor.pub`
+
+func GetRekorPub() string {
```

Comment:
IMHO we should put this somewhere under `cli/` in order to keep logging and `cosign` binary-specific out of package

Reply:
Ohhh hmmmm you mean keeping `fmt.Println` out? It's used elsewhere in this file as well

Reply:
hrm yeah